internal/validator/validator.go

```diff
@@ -116,6 +116,27 @@ func checkMessageSubject(s *string) error {
 	return nil
 }
 
+/**
+ * checkMessageLength - Checks if the commit message is equal to or under 50 characters.
+ *
+ * This function ensures the commit message does not exceed 50 characters in length.
+ *
+ * Parameters:
+ *  - l (*string): A pointer to the string representing the length
+ *				   of the commit message.
```

Comment:
The parameter name `l` is misleading, how about something like `msg` since it is synonymous to "commit message" which this function should be checking for its length? Update the docs once you have made the necessary changes as well.